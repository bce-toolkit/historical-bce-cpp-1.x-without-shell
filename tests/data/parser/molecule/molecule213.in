(Co(NH3)4CO3)2SO4.3H2O
