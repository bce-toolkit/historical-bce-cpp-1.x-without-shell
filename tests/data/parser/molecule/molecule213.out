C 2
Co 2
H 30
N 8
O 13
S 1
